src/dlgTriggerEditor.cpp

```diff
@@ -5035,6 +5035,7 @@ void dlgTriggerEditor::slot_trigger_selected(QTreeWidgetItem* pItem)
         mpTriggersMainArea->spinBox_lineMargin->setValue(pT->getConditionLineDelta());
         mpTriggersMainArea->spinBox_stayOpen->setValue(pT->mStayOpen);
         mpTriggersMainArea->groupBox_soundTrigger->setChecked(pT->mSoundTrigger);
+        mpTriggersMainArea->lineEdit_soundFile->setToolTip(pT->mSoundFile);
```

Comment:
:-1: What about if there is no sound file selected? There will be no tooltip (though there is the placeholder text instead) - but the user cannot type anything into this `QLineEdit` as it is a *read-only* one, they may eventually realise this but probably not the first time...

Reply:
![image](https://user-images.githubusercontent.com/110988/61585961-60797880-ab69-11e9-8386-e454e8ebc3d5.png)